### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by thaJeztah**
carry https://github.com/docker/docker/pull/21825

closes https://github.com/docker/docker/pull/21825

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @tiborvass @mlaventure  :smile:

**Comment by tiborvass** [comment]
LGTM

**Comment by vieux** [comment]
SGTM

**Comment by crosbymichael** [comment]
LGTM

**Comment by thaJeztah** [comment]
author LGTM no CI needed on this one

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,6 +77,9 @@ be found.
 
 ### Networking
 
+- Fix panic if a node is forcibly removed from the cluster ([#21671](https://github.com/docker/docker/pull/21671))
+- Fix "error creating vxlan interface" when starting a container in a Swarm cluster ([#21671](https://github.com/docker/docker/pull/21671))
+- Fix `host` and `none` networks to disappear until restart when SIGHUP is used to reload cluster configuration ([#21671](https://github.com/docker/docker/pull/21671))
```
Comment (cpuguy83) [inline CHANGELOG.md]: I don't think this needs to be in there.
We didn't support reloading the cluster config in 1.10

File: CHANGELOG.md [hunk 2]
```diff
@@ -92,6 +97,12 @@ be found.
 
 ### Runtime
 
+- Fix a panic that could occur when cleanup after a container started with invalid parameters ([#21716](https://github.com/docker/docker/pull/21716))
+- Fix a race with event timers stopping early ([#21692](https://github.com/docker/docker/pull/21692))
+- Fix race conditions in the layer store, potentially corrupting the map and crashing the process ([#21677](https://github.com/docker/docker/pull/21677))
+- Un-deprecate auto-creation of host directories for mounts. This feature was marked deprecated in ([#21666](https://github.com/docker/docker/pull/21666))  
+  Docker 1.9, but was decided to be too much of an backward-incompatible change, so it was decided to keep the feature.
+- Fix an issue where parallel aufs mount calls produce invalid argument error ([#21552](https://github.com/docker/docker/pull/21552))
```
Comment (cpuguy83) [inline CHANGELOG.md]: No need for this one since the 1.10 behavior didn't allow parallel calls to aufs.

**Comment by thaJeztah** [comment]
@cpuguy83 thanks! I'll open a PR to remove those

**Pull request merged**